api/go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/gin-contrib/sessions v0.0.3
 	github.com/gin-contrib/static v0.0.0-20200916080430-d45d9a37d28e
 	github.com/gin-gonic/gin v1.6.3
+	github.com/go-playground/assert/v2 v2.0.1
```

Comment:
why do we need to add this library? 

I think that we can only update `github.com/tidwall/gjson` is enough 

@nic-chen please confirm

Reply:
@ThanoshanMV  we don't need this lib, please remove it. thanks.

Reply:
I'll remove it.